apps/coscrad-frontend/src/components/resources/shared/resource-preview.tsx

```diff
@@ -0,0 +1,16 @@
+import { AggregateType } from '@coscrad/api-interfaces';
+
+interface ResourcePreviewProps {
+    resourceType: AggregateType;
+}
+
+const mediaUrl = 'https://kaaltsidakah.net/coscrad/resource-icons';
+const iconDimension = '80px';
```

Comment:
Seems like this will need to be hard-wired. How will we resize custom images? Or will custom images be validated to be the right size? Eventually, we may want `mediaAssetID` instead of public URLs.